src/python/pants/backend/url_handlers/s3/register.py

```diff
@@ -18,14 +18,15 @@
 from pants.util.strutil import softwrap
 
 CONTENT_TYPE = "binary/octet-stream"
-
+EMPTY_SHA256_HASH = "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
```

Comment:
FYI you can get this value from from `pants.engine.fs.EMPTY_DIGEST` or `pants.engine.fs.EMPTY_FILE_DIGEST`.

Reply:
Aren't those of type `Digest / FileDigest` though? I need it as a string for the header. I might be able to do something like 

```
empty_digest = await Get(Digest, Digest, EMPTY_FILE_DIGEST)

headers.add_header(
        "X-Amz-Content-SHA256",
        empty_digest.fingerprint,
    )
```

But seems unnecessarily complicated?

Reply:
My thought was to just reference `EMPTY_FILE_DIGEST.fingerprint` which is a string.

Reply:
thanks, I confirmed this works